SCClassLibrary/Common/Collections/Array.sc

```diff
@@ -131,6 +131,14 @@ Array[slot] : ArrayedCollection {
 		_ArrayContainsSeqColl
 		^this.primitiveFailed
 	}
+	maxDepth { arg max = 1;
+		_ArrayMaxDepth
+		^super.maxDepth(max)
+	}
+	maxSizeAtDepth { arg rank;
+		_ArrayMaxSizeAtDepth
+		^super.maxSizeAtDepth(rank)
+	}
```

Comment:
I can add them, but they are not strictly required.

 If we decide that this should be a standard, then there would be many places for new newlines in the class library.